src/content/docs/browser/new-relic-browser/troubleshooting/google-indexing-unknown-paths.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Google Indexing 404 Paths
+type: troubleshooting
+tags:
+  - Browser
+  - Browser monitoring
+  - Troubleshooting
+metaDescription: 'Google may attempt to index invalid site paths from strings found in the agent loader.'
+---
+
+## Problem
+
+You're seeing 404 errors in your Google Search dashboard for URLs that are not valid for your site. You may also see 404 errors in your web server logs for these paths. After searching your site, you found the site path is present as a string in the Browser Agent loader script being injected into your HTML.
+
+## Cause
+
+The Browser Agent makes use of string literals in code but converts those to string concatenation during the build process to prevent possible issues with code that may wrap the loader string in a string literal. In some cases, this may lead to strings the start with a forward slash `/`. When Google indexes the page containing the agent loader script, it will store these strings as potential paths for your site and attempt to index them.
+
+## Solutions
+
+This is not an uncommon scenario for site administrators and is not just linked to the Browser Agent. There could be many causes for strings to appear in your site HTML that Google Search may perceive as a potential path within your site. See the below resources of other people having this same concern.
+
+- [How can I block Google from crawling things it thinks are URLs from __NEXT_DATA__?](https://stackoverflow.com/questions/75672103/how-can-i-block-google-from-crawling-things-it-thinks-are-urls-from-next-data)
```

Comment:
How stable are stackoverflow links?  Can we reasonably trust these won't go "dead"?

Reply:
I would not worry about it. SO is pretty stable and doesn't seem to ever delete anything.